packages/pkcs12/src/__tests__/index-test.ts

```diff
@@ -39,7 +41,29 @@ const uppercaseAliasKeystoreP12 = {
   password: 'password',
   alias: 'UPPERCASE',
 };
-describe('reading PKCS#12 files', () => {
+
+/**
+ * Generate DSA certificate:
+ * openssl dsaparam -out params.pem 3072
+ * openssl gendsa -out key.pem params.pem
+ * openssl req -new -key key.pem -out req.pem
+ * openssl x509 -req -in req.pem -signkey key.pem -out certificate.cer
+ *
+ * Package into a PKCS#12 keystore:
+ * keytool -import -alias test -file certificate.cer -keystore dsapkcs.keystore -storetype pkcs12
+ *
```

Comment:
How are the fingerprints calculated?

Reply:
fingerprints are calculated by getting the asn.1 representation of the certificate, then hashing it.